onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -663,18 +661,30 @@ static Status CopySparseData(size_t n_sparse_elements,
   Status status = Status::OK();
   TensorShape indices_shape(indices.dims().data(), indices.dims().size());
 
-  auto indices_data = gsl::make_span<const int64_t>(indices.int64_data().data(), static_cast<size_t>(indices_shape.Size()));
+  ORT_RETURN_IF_NOT(indices.data_type() == ONNX_NAMESPACE ::TensorProto_DataType_INT64, "Indicies expected to be INT64");
+
+  gsl::span<const int64_t> indicies_data;
+  const auto elements = static_cast<size_t>(indices_shape.Size());
```

Comment:
nit: it's 'indices' isn't it? [https://www.dictionary.com/browse/indices](https://www.dictionary.com/browse/indices)